test/functional/module_stream/info_test.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../test_helper'
+require 'hammer_cli_katello/module_stream'
+
+module HammerCLIKatello
+  describe ModuleStreamCommand::InfoCommand do
+    it 'allows ID' do
```

Comment:
it 'resolves ID from name and product ID'
it 'resolves ID from name, product name, and repository ID'

Reply:
unfortunately, search for module streams isn't implemented, so I'm having trouble testing these options as they don't work quite right as is. We can delay this PR until search gets implemented, or add tests later.